src/fetch_real_estate_prices.py

```diff
@@ -0,0 +1,126 @@
+import json
```

Comment:
Disclaimer: 💅

Python style guide propose [a convention to organize the `import` section](https://www.python.org/dev/peps/pep-0008/#imports). In a nutshell first we have native modules, then modules installed via `pip`, and then your application's modules. And on top of that, `import` comes before `from … import …`.

That said, I would have written:

```python
import json
import os

import numpy as np
import pandas as pd
import requests
from geopy.geocoders import GoogleV3
```

But… well… again: 💅